task_templates/transforms/python3_image_transform/model-metadata.yaml

```diff
@@ -0,0 +1,35 @@
+name: python3_anomaly_detection
+type: training
+targetType: transform
+# modelID: optional
+environmentID: 5e8c889607389fe0f466c72d
+#trainingModel:
+#  trainOnProject: optional project on which to train
+validation:
+  # Path is relative to this file
+  input: ../../../tests/testdata/cats_dogs_small_training.csv
+  targetName: class
+typeSchema:
+  input_requirements:
+    - field: data_types
+      condition: EQUALS
+      value: IMG
+    - field: number_of_columns
+      condition: NOT_LESS_THAN
+      value: 1
+    - field: sparse
+      condition: EQUALS
+      value: FORBIDDEN
+    - field: contains_missing
+      condition: EQUALS
+      value: FORBIDDEN
+  output_requirements:
+    - field: data_types
+      condition: EQUALS
+      value: IMG
+    - field: sparse
+      condition: EQUALS
+      value: NEVER
+    - field: contains_missing
+      condition: EQUALS
+      value: NEVER
```

Comment:
Are these the defaults? Should we have them in the metadata?

Reply:
The default is supported.